src/legacy/core_plugins/kibana/public/discover/np_ready/angular/context_state.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  createStateContainer,
+  createKbnUrlStateStorage,
+  syncStates,
+} from '../../../../../../../plugins/kibana_utils/public';
+import { Filter } from '../../../../../../../plugins/data/common/es_query/filters';
```

Comment:
Import from plugin should be from root.
Not sure why ESLint didn't argue. If I am not mistaken @flash1293 was fixing something like this in dashboard https://github.com/elastic/kibana/pull/55818

this particular import should be like:

```
import { esFilters } from '../../../../../../../plugins/data/public';

esFilters.Filter
```

Reply:
thx, fixed that!